gengokit/httptransport/httptransport.go

```diff
@@ -24,6 +24,8 @@ import (
 type Helper struct {
 	Methods           []*Method
 	PathParamsBuilder string
+	ServerTemplate    func(interface{}) (string, error)
+	ClientTemplate    func(interface{}) (string, error)
```

Comment:
They do not operate on the struct or use any information from the struct so they are func fields of the struct rather than methods of the struct.